src/SDKs/_metadata/datafactory_resource-manager.txt

```diff
@@ -1,11 +1,10 @@
-2018-02-21 00:11:46 UTC
+2018-03-05 03:34:00 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      76df6d1167d0943f523747e9303d3dfd6f817e00
```

Comment:
Please pull down changes from upstream and run `generate.cmd` again, this should add the commit id here.

Reply:
@hvermis @najian you still need to log the commit id here before we can merge